src/main/resources/DB/init.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE Homework
+(
+id 		integer GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+name 	VARCHAR(40) NOT NULL,
+description VARCHAR(200)
+);
+CREATE TABLE Lesson
+(
+id		integer GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+name	VARCHAR(40) NOT NULL,
+updatedAt TIMESTAMP DEFAULT NOW(),
+homework_id INTEGER,
+FOREIGN KEY (homework_id) REFERENCES Homework(id)
+);
+CREATE TABLE Schedule
+(
+id		integer GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+name	VARCHAR(40) NOT NULL,
+updatedAt TIMESTAMP DEFAULT NOW()
+);
+CREATE TABLE Lessons
```

Comment:
```suggestion
CREATE TABLE Lesson_Schedule
```
також часто з'єднують назви, щоб було розуміння, що це допоміжна таблиця